pino.js

```diff
@@ -30,8 +32,15 @@ function pino (opts, stream) {
     istream = iopts
     iopts = defaultOptions
   }
-  istream = istream || process.stdout
   iopts = Object.assign({}, defaultOptions, iopts)
+  if (iopts.extreme && iopts.prettyPrint) throw Error('cannot enable pretty print in extreme mode')
+  istream = istream || process.stdout
+  if (iopts.prettyPrint) {
+    var pstream = pretty(iopts.prettyPrint)
+    var origStream = istream
+    pstream.pipe(origStream)
```

Comment:
Done. But I left off an error function because we are not within an actual instance at that point. Thus, we can't emit the error.